arvo/jael/scry.md

```diff
@@ -0,0 +1,222 @@
++++
+title = "Scry Reference"
+weight = 3
+template = "doc.html"
++++
+
+Jael's scry endpoints never take a `care`. The particular endpoints are specified in the place of the desk in the path prefix, then additional arguments are specified in the path.
```

Comment:
Wondering if we should call this out as being considered old-style scry paths, considering all recent additions (in ames, dill, eyre) do use cares and expect an empty desk.